readme.md

```diff
@@ -1,7 +1,7 @@
-[![Build Status](https://travis-ci.org/mightyiam/eslint-config-standard-with-typescript.svg?branch=master)](https://travis-ci.org/mightyiam/eslint-config-standard-with-typescript)
+[![Build Status](https://travis-ci.org/standard/eslint-config-standard-with-typescript.svg?branch=master)](https://travis-ci.org/standard/eslint-config-standard-with-typescript)
```

Comment:
Why doesn't this image load?

Reply:
Loads for me! Says 'Build Failing'. It might be cached by your browser.